docs/docs/rest-api/public/api/v2/events.raml

```diff
@@ -16,6 +16,11 @@ get:
         description:
             Specify subscribed event types.
             You can specify this parameter multiple times with different values.
+    plan-format:
```

Comment:
Is this part of this PR? Could it be that you meant to include the `reason` field?

Reply:
This is a clean up. Boy Scout rule. We never followed up.

Reply:
It could be a separate PR, but I am totally fine with it being here 👍